modules/registrars/openprovider/openprovider.php

```diff
@@ -329,6 +330,50 @@ function openprovider_ResendIRTPVerificationEmail(array $params)
     return openprovider_registrar_launch_decorator('resendIRTPVerificationEmail', $params);
 }
 
+
+function openprovider_config_validate($params)
+{
+    $username = $params['Username'];
+    $password = $params['Password'];
+    $test = $params['test_mode'];
+    $ip = $_SERVER['SERVER_ADDR'];
+
+    $valid = false;
+
+    if ($test == 'on') {
+        $env = 'Sandbox';
+        $url = 'http://api.sandbox.openprovider.nl:8480/v1beta/auth/login';
```

Comment:
make these values as constant in the file atleast, or if we have any config files that's better

Reply:
These changes have been added